src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a href="#a" class="nav__link is-active">apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#s" class="nav__link">samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#сы" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="list__item" data-qa="hover">
+            <a href="#н" class="nav__link">ноутбуки и компьютеры</a>
+          </li>
+          <li class="list__item">
+            <a href="#г" class="nav__link">гаджеты</a>
+          </li>
+          <li class="list__item">
+            <a href="#п" class="nav__link">планшеты</a>
+          </li>
+          <li class="list__item">
+            <a href="#ф" class="nav__link">фото</a>
+          </li>
+          <li class="list__item">
+            <a href="#в" class="nav__link">видео</a>
```

Comment:
don't use ru letters in `href` use `en` you may leave `#` in hrefs here or use semantic names `phones`, `tabs`, `photo` etc

Reply:
corrected